packages/botonic-cli/templates/intent/tests/app.test.js

```diff
@@ -13,7 +16,8 @@ const o = new BotonicOutputTester(app)
 test('TEST: (404) NOT FOUND', async () => {
   await expect(i.text('whatever')).resolves.toBe(
     o.text(
-      `Enter the generated JSON key for dialogflowV2 in plugins.js to test the bot. Try typing "hello" to start the bot.`
+      // replace with  'Try typing "hello" to start the bot.' after configuring dialogflow
```

Comment:
did you forget to remove it?

Reply:
no, the expected text was wrong (it was concatenating the normal message and the one which appears when dialogflow is not configure